src/main/java/io/jenkins/plugins/view/calendar/service/CronWrapper.java

```diff
@@ -0,0 +1,73 @@
+package io.jenkins.plugins.view.calendar.service;
+
+import hudson.scheduler.CronTab;
+import io.jenkins.plugins.extended_timer_trigger.CronTabWrapper;
+import java.time.ZonedDateTime;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import org.jenkinsci.plugins.parameterizedscheduler.ParameterizedCronTab;
+
+public abstract class CronWrapper<T> {
```

Comment:
This is a wrapper around the crontabs. The extended timer trigger plugin has not the crontabs from core but due to it's support for the Jenkins crontab and the extended syntax a wrapper.
So I need here as well a wrapper to unify access to the required methods